bugsnag-android-core/src/test/java/com/bugsnag/android/NativeInterfaceApiTest.kt

```diff
@@ -220,11 +235,22 @@ internal class NativeInterfaceApiTest {
     }
 
     @Test
-    fun notifyCall() {
-        NativeInterface.notify("SIGPIPE", "SIGSEGV 11", Severity.ERROR, arrayOf())
+    fun notifyJVMStackTraceCall() {
+        NativeInterface.notify(
+            "SIGPIPE",
+            "SIGSEGV 11",
+            Severity.ERROR,
+            arrayOf<StackTraceElement>()
+        )
         verify(client, times(1)).notify(any(), any())
     }
 
+    @Test
+    fun notifyNativeTraceCall() {
+        NativeInterface.notify("SIGPIPE", "SIGSEGV 11", Severity.ERROR, arrayOf<NativeStackframe>())
+        verify(client, times(1)).populateAndNotifyAndroidEvent(any(), any())
```

Comment:
perhaps there's something more specific to be verified here, like to ensure the event was created with the correct error class / message?